args/src/lib.rs

```diff
@@ -144,8 +233,17 @@ impl CmdArgs {
     pub fn kernel_num_workers(&self) -> usize {
         self.num_workers.into()
     }
+    // backwards-compatible, to deprecate
```

Comment:
Do we need to be backward-compatible at this point? Knowing we still haven't cut a release, it's probably a good time to remove it?

Reply:
It would be trivial to modify the code to use the new method. However, we have changes in the kernel driver in other branches (which are going to severely conflict) and making that change would just add more noise.